samples/bluetooth/fast_pair/input_device/README.rst

```diff
@@ -458,12 +458,13 @@ Personalized Name extension
 Testing Personalized Name extension is described in `Fast Pair Certification Guidelines for Personalized Name`_.
 
 .. note::
-   To mitigate Android Personalized Name write issues, whenever you change the Personalized Name on an Android phone, perform the following:
+   For Android devices running Android 15 or lower, to mitigate Android Personalized Name write issues, whenever you change the Personalized Name on an Android phone, perform the following:
 
    * Write the new Personalized Name.
```

Comment:
Aren't these steps that should be performed in this order? If yes, use a numbered list.